backends/aerospike_test.go

```diff
@@ -89,8 +89,8 @@ func (c *goodAerospikeClient) Put(policy *as.WritePolicy, aeKey *as.Key, binMap
 	return as_types.NewAerospikeError(as_types.KEY_MISMATCH)
 }
 
-func (c *goodAerospikeClient) NewUuidKey(namespace string, key string) (*as.Key, error) {
-	return as.NewKey(namespace, setName, key)
+func (c *goodAerospikeClient) NewUuidKey(namespace string, set string, key string) (*as.Key, error) {
+	return as.NewKey(namespace, set, key)
 }
 
 func TestNewAerospikeBackend(t *testing.T) {
```

Comment:
I notice that our tests are not updated, and we are thus passing a config with an empty `Set` value. Should we update the tests to have a non-null value, and perhaps modify the code to throw an error if the value is missing?

Reply:
Good point, I'll add some extra validation around this.
Although this should never be empty in actual run, as it defaults on missing to the old hardcoded constant.

Reply:
Are you planning on adding this validation soon?

Reply:
I was waiting for this PR https://github.com/prebid/prebid-cache/pull/81 to be merged first.
As these two will have conflicts.

I'll close this for now and reopen it in the future.